Dockerfile

```diff
@@ -17,6 +17,9 @@ RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -o manager main.
 
 FROM alpine:3.15
 
+# Upgrade for security patches.
+RUN apk update && apk add --upgrade zlib-dev
```

Comment:
Do we want to upgrade everything in Alpine? (Is zlib-dev updated in 3.15?)
Any reason not to move to 3.16?

Reply:
We've been fiddling trying to verify what fix could be applied. 3.15 actually uses zlib 3.2.12-r3 so its not clear why quay is kicking up a fuss.